tensorboard/webapp/feature_flag/effects/feature_flag_effects.ts

```diff
@@ -33,6 +34,7 @@ export class FeatureFlagEffects {
       combineLatestWith(this.store.select(getIsAutoDarkModeAllowed)),
       map(([, isDarkModeAllowed]) => {
         const features = this.dataSource.getFeatures(isDarkModeAllowed);
+        features.forceSVG = this.ForceSVGDataSource.getAndUpdateForceSVGFlag();
```

Comment:
This all sounds good. However, if we are only pulling from localStorage for the ForceSVGDataSource it does not need all the overhead of an injectable class. I am going to change it to force_svg_util and just have a getForceSvg and updateForceSvg functions as standalone functions. Does that sound good to you?

Reply:
In this particular case, I can empathise with the temptation to avoid using Angular's Dependency Injection (DI). Why you might think this way:
1. force_svg_util itself currently has no DI-able dependencies (although I think it's hard to be certain that will be true forever)
2. we don't foresee wanting to to DI some other implementation of ForceSVGDataSource -- we think it will always be a local storage thing, unlike persistent settings which might one day be stored on the server.

Having said that, I think it's good habit/hygiene to make this a DI-able object like any other object in our system that reads/writes to a data source. I agree there is some overhead but it is pretty standard Angular stuff. It will also give you standard options/patterns when you write tests - how to Mock, how to Fake/Stub.

Reply:
I made it injectable again. 

I also added tests.

PTAL.